pkg/manager/member/executor_scaler.go

```diff
@@ -1 +1,186 @@
 package member
+
+import (
+	"context"
+	"fmt"
+	"github.com/pingcap/tiflow-operator/api/v1alpha1"
+	"github.com/pingcap/tiflow-operator/pkg/manager/member/prune"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	autoscaling "k8s.io/api/autoscaling/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+type executorScaler struct {
+	ClientSet kubernetes.Interface
+	PVCPruner prune.PVCPruner
+}
+
+// NewExecutorScaler return a executorScaler
+func NewExecutorScaler(clientSet kubernetes.Interface) Scaler {
+
+	return &executorScaler{
+		ClientSet: clientSet,
+		PVCPruner: nil,
```

Comment:
How about initiailizing `PVCPruner` here?

Reply:
Here an instance of `tc` is needed to build the new PVCPruner structure, and, I think the logic should only be processed when the upper layer needs to scale

Reply:
Here an instance of `tc` is needed to build the new PVCPruner structure, and, I think the logic should only be processed when the upper layer needs to scale

Reply:
I think we can keep these logic in PVCPruner but add `tc` as an argument of `PVCPruner.Prune`. Current logic actually can't support concurrently execute `Scale` for several `TiflowClusters`(PVCPruner will conflict.)

Reply:
This can also make sure the logic is only be processed when the upper layer needs to scale

Reply:
Okay, I get it.